tests/src/devdox_ai_git/test_repo_fetcher.py

```
@@ -45,6 +77,136 @@ def get_user_repositories(self, *args, **kwargs):
+                "path": kwargs.get("file_path"),
+                "message": kwargs.get("commit_message")
+            }
+
+    class SpyGitHubManager:
+        """Drop-in replacement for rf.GitHubManager used inside GitHubRepoFetcher."""
+
+        def __init__(self, base_url):
+            self.base_url = base_url
+            self.last_auth = None
+
+        def authenticate(self, token: str):
```

Ruff (0.13.2)

Unused method argument: `token`

(ARG002)